libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/location/LocationEngineConductor.java

```diff
@@ -31,11 +31,11 @@ public void onDestroy() {
     deactivateLocationEngine();
   }
 
-  public void initializeLocationEngine(Context context, boolean simulateRoute) {
-    initLocationEngine(context, simulateRoute);
+  public void initializeLocationEngine(Context context, boolean shouldReplayRoute) {
+    initLocationEngine(context, shouldReplayRoute);
   }
 
-  public void updateRoute(DirectionsRoute route) {
+  public void updateSimulatedRoute(DirectionsRoute route) {
```

Comment:
Let's make sure we update the release notes and the android docs site with these changes, as we are breaking SemVer here 👍